SimpleWAWS/Controllers/ResourceController.cs

```diff
@@ -23,9 +23,10 @@ public class ResourceController : ApiController
         private static int _userGotErrorErrorCount = 0;
         public async Task<HttpResponseMessage> GetResource()
         {
-            var resourceManager = await ResourcesManager.GetInstanceAsync();
-            var resourceGroup = await resourceManager.GetResourceGroup(HttpContext.Current.User.Identity.Name);
-            return Request.CreateResponse(HttpStatusCode.OK, resourceGroup == null ? null : resourceGroup.UIResource);
+         var resourceManager = await ResourcesManager.GetInstanceAsync();
+         var resourceGroup = await resourceManager.GetResourceGroup(HttpContext.Current.User.Identity.Name);
+
+            return Request.CreateResponse(HttpStatusCode.OK, resourceGroup == null ? null : (HttpContext.Current.Request.QueryString["appServiceName"] == "Function")? resourceGroup.FunctionsUIResource: resourceGroup.UIResource);
```

Comment:
You mean revert the extra line /indentation? The `FunctionsUIResource` is still needed for TryFunctions portal